pkgs/development/python-modules/whisperx/default.nix

```
@@ -0,0 +1,63 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+
+  setuptools,
+
+  ctranslate2,
+  faster-whisper,
+  ffmpeg,
+  nltk,
+  pandas,
+  pyannote-audio,
+  torch,
+  torchaudio,
+  transformers,
+}:
+buildPythonPackage rec {
+  pname = "whisperx";
+  version = "3.3.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "m-bain";
+    repo = "whisperX";
+    tag = "v${version}";
+    hash = "sha256-Wb9jKTs0rTdgjvcnKl1P8Uh6b5jzlkKgQBcJYA+7+W4=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    ctranslate2
+    faster-whisper
+    nltk
+    pandas
+    pyannote-audio
+    torch
+    torchaudio
+    transformers
+  ];
+
+  pythonRelaxDeps = [
+    "ctranslate2"
+    "faster-whisper"
+  ];
+
+  pythonImportsCheck = [ "whisperx" ];
+
+  # No tests in repository
+  doCheck = false;
+
+  makeWrapperArgs = [ "--prefix PATH : ${lib.makeBinPath [ ffmpeg ]}" ];
+
+  meta = {
+    mainProgram = "whisperx";
+    description = "Automatic Speech Recognition with Word-level Timestamps (& Diarization) ";
+
```

```suggestion
  homepage = "https://github.com/m-bain/whisperX";
  changelog = "https://github.com/m-bain/whisperX/releases/tag/v${version}";
```
Please remove useless blank line and add homepage info.